app/src/test/java/invoice_automation/module/QuickBooksModuleTest.java

```diff
@@ -191,5 +209,76 @@ public void testGetInvoiceWithMatchingMemo_noMatchingInvoice() throws Exception
         // Verify
         assertNull(returnedInvoice);
     }
+
+    public Customer getExpectedCustomer() {
```

Comment:
nit: if we're using the same customer, just initialize and set it as a class variable like we did with the address and school

Reply:
I changed the variable to the class variable "customer", but I used the method "setupCustomer" to instantiate it since Customer doesn't have a constructor or builder method as far as I'm aware. Lmk if it should be changed